pyright/alt-1/requirements.txt

```diff
@@ -0,0 +1,104 @@
+### Requirements: alt-1
+#
+# This file defines the dependencies for an alternative pyright environment. The packages listed in
+# this file will not be present in the environment that pyright/pylance will use by default when
+# running as a language server.
+#
+# NOTE: It is important that dagster-* dependencies of any package listed here are also listed.
+# e.g. if examples/foo depends on python_modules/libraries/dagster-foo, then dagster-foo must be
+# listed if examples/foo is listed.
+#
+# NOTE: Paths are relative to repo root.
```

Comment:
what is the plan for keeping this file up to date?

Reply:
There is no special plan-- I think it can more-or-less be treated like any one of our gazillion toxfiles, which also have deps that have to be kept in sync with corresponding setup.py files.

I would love it if we could get everything in one env or there were some kind of more automated system for partitioning our packages into envs, but I'm not aware of a better option.